docs/release-notes/deepspeed-cpu-offloading.txt

```diff
@@ -0,0 +1,5 @@
+:orphan:
+
+**New Features**
+
+-  Example: DeepSpeed ZeRO Stage 3 with CPU offloading.
```

Comment:
suggested edit: Train models that do not fit into GPU memory using DeepSpeed ZeRO Stage 3 with CPU offloading.